<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="equation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: skyblue
</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>630</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forza</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { color: red }</string>
   </property>
   <property name="text">
    <string>MathleCommand</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>115</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>label</zorder>
  <zorder>answerBox</zorder>
  <zorder>score_label</zorder>
  <zorder>equation</zorder>
  <zorder>title</zorder>
  <zorder>score</zorder>
  <zorder>answerButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>answerBox</tabstop>
  <tabstop>answerButton</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
